realm/realm-library/src/androidTest/java/io/realm/SortTest.java

```diff
@@ -153,19 +153,19 @@ private void checkSortTwoFieldsStringAscendingIntAscending(RealmResults<AllTypes
 
         assertEquals("Adam", results.get(0).getColumnString());
         assertEquals(4, results.get(0).getColumnLong());
-        assertEquals(2, ((TableView) results.getTable()).getSourceRowIndex(0));
+        assertEquals(2, ((TableView) results.getTableOrView()).getSourceRowIndex(0));
```

Comment:
UH... i actually have the same question .... if we use `Results` from OS, the `TableView` and strange `TableOrView` are going to be removed ..